### ISSUE TITLE
Select control issue- Invalid aria attributes (Accessibility)

### ISSUE THREAD

**Issue by yuvrajsawant-github** [ant-design/ant-design#24406]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/iql7n)

### Steps to reproduce
1. Navigate to sandbox
2. Open the page in chrome 
3. Run the Accessibility insights for web - fast pass (Chrome plug in)
4. Check the reported issue related to aria attributes
5. Please refer attached html file and screen shot

### What is expected?
On load application the select issue should not be reported in the fast pass, its causing accessibility issues for the users with disability.

### What is actually happening?
1. Errors related to aria attributes are displayed in fast pass
2. Narrator unable to indicate the user to open the select due to missing attributes

| Environment | Info |
|---|---|
| antd | 4.2.4 |
| React | 16.12.0 |
| System | 16.12.0 |
| Browser | Edge Version 81.0.416.72 (Official build) (64-bit) or Chorme Version 81.0.4044.138 (Official Build) (64-bit) |

**Comment by yuvrajsawant-github** [comment]
[files.zip](https://github.com/ant-design/ant-design/files/4669984/files.zip)
Please refer attached files for details.

**Comment by yoyo837** [comment]
ref: #16270

**Comment by zombieJ** [comment]
Open the Select and retry. Current report is not correct since associated element is not created before operating on.

And for `aria-label` is user pass props. I will add in input also.
